app/views/submissions/_assigned_evaluators.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="usa-accordion">
+  <% rand = SecureRandom.hex(8) %>
+  <h2 class="usa-accordion__heading" id="<%= "#{rand}-accordion-heading" %>">
+    <button
+      type="button"
+      class="usa-accordion__button display-flex flex-justify icon-white"
+      aria-expanded="true"
+      aria-controls="<%= rand %>"
+    >
+      <div>Assigned Evaluators</div>
+    </button>
+  </h2>
+
+  <div class="usa-accordion__content" id="<%= rand %>">
+      <table class="usa-table usa-table--stacked-header usa-table--borderless gray-header">
+          <thead>
+            <tr>
+              <th scope="col">Evaluator name</th>
+              <th scope="col">Evaluation Status</th>
+              <th scope="col">Score</th>
+              <th scope="col" aria-label="Actions"></th>
+            </tr>
+          </thead>
+          <tbody>
+            <% @submission.evaluators.each do |evaluator| %>
+              <tr data-evaluator-id="<%= evaluator.id %>" data-evaluator-type="user">
+                <th scope="row" data-label="Evaluator name" class="text-top">
+                  <span class="text-bold text-ls-1 line-height-sans-4"><%= "#{evaluator.first_name} #{evaluator.last_name}" %></span>
+                </th>
+                <td data-label="Evaluation Status" class="text-top"></td>
+                <td data-label="Score" class="text-top"></td>
+                <td data-label="">
+                  <div class="display-flex flex-no-wrap grid-row">
+                    <%= form_with url: phase_evaluator_submission_assignments_path(@submission.phase) do |f| %>
+                      <%= f.hidden_field :evaluator_id, value: evaluator.id %>
+                      <%= f.hidden_field :submission_id, value: @submission.id %>
+                      <%= f.submit "Unassign" %>
```

Comment:
this form routes to the create action when it should issue an update to the existing evaluator_submission_assignment to mark it unassigned if it's assigned and recused_unassigned if it is recused status.
`assigned` => `unassigned`
`recused` => `recused_unassigned`
because it creates a new evaluator_submission_assignment, that also creates duplicate assignment records. we should also validate uniqueness on the combination of evaluator_id/phase_id/submission_id in the EvaluatorSubmissionAssignment model, so we don't end up with duplicate records (just in case someone tries to create one).